### PR TITLE
Week 1 HW

### PR DESCRIPTION
**Pull request by psytau**
Here's the homework!

### CODE REVIEW

**Comment by psytau** [comment]
Should pull request be to the master or the answers branch?

**Comment by reneedv** [comment]
Either works, but master is fine.

On Thursday, January 16, 2014, psytau notifications@github.com wrote:

> Should pull request be to the master or the answers branch?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/UWE-Ruby/RubyWinter2014/pull/32#issuecomment-32482622
> .

## 

Thanks!
**Renée

President, NIRD LLC
http://nird.us
@NIRDLLC
707.266.NIRD

File: week1/homework/questions.txt [hunk 1]
```diff
@@ -3,13 +3,25 @@ Chapter 3 Classes, Objects, and Variables
...
+  There are instance variables within objects (@x or @y) which persist in the object. Then there are local variables without a funny prefix. There are $GLOBAL_VARIABLES with a $ prefix. There are variables with two @@ that are associated with the class. Even a class name is a variable, I suppose. They should be capitalized (Klass).
 
 3. What is the difference between an object and a class?
+  An object is an instance of a class. (But a Class is also an Object, technically.) a class is for creating new objects.
 
 4. What is a String?
+  An object or, I guess Class that creates objects, which hold text, and has handy methods for operating on text.
 
 5. What are three messages that I can send to a string object? Hint: think methods
+  "encoding", "length", "+" (does that count?)
```
Comment (reneedv) [inline week1/homework/questions.txt:22]: yes operators like + are methods in Ruby

**Comment by reneedv** [comment]
Good Work!

**Comment by reneedv** [comment]
Hi All,

  I think you will get an email from UW tomorrow about class on Thursday,
but here is the link for the Adobe connect classroom:

http://uweoconnect.extn.washington.edu/ruby210/

Your UW credentials should have access. Please email me if you have any
problems:
renee@nird.us

Thanks!
**Renée

President, NIRD LLC
http://nird.us
@NIRDLLC
707.266.NIRD

**Comment by reneedv** [comment]
Hi All,

  If you have not sent me your final project email, please do so, they were
due 20 minutes ago.

  Thanks for a great class! Please feel free to reach out if you have
further questions or need help or guidance!

Thanks!
**Renée

President, NIRD LLC
http://nird.us
@NIRDLLC
707.266.NIRD

**Comment by Mattsface** [comment]
IT's coming Renee!!

On Wed, Sep 3, 2014 at 12:23 AM, Renée Hendricksen <notifications@github.com

> wrote:
> 
> Hi All,
> 
> If you have not sent me your final project email, please do so, they were
> due 20 minutes ago.
> 
> Thanks for a great class! Please feel free to reach out if you have
> further questions or need help or guidance!
> 
> Thanks!
> **Renée
> 
> President, NIRD LLC
> http://nird.us
> @NIRDLLC
> 707.266.NIRD
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/UWE-Ruby/RubyWinter2014/pull/32#issuecomment-54260801
> .

## 
- Matthew Spah
  _Implementation Engineer_ for Coalfire Systems Inc
  _Computer Information Sciences_ _Tutor_ for Everett Community College
  _Network Specialist & Web Admin_ for the Xenapi Admin Project
  http://xenapiadmin.com
  CompTIA Security+ Certified and Cisco CCNA Certified
  Arlington, WA Cell 425-319-8676

_"What if everything around you. Isn't quite as it seems?" - *_Trent Reznor*

*"Experience is the name everyone gives to their mistakes." -- Oscar Wilde *

**Comment by Mattsface** [comment]
Email was sent to  Renée Hendricksen renee@nird.us

Titled:   Fwd: Matt Spah and Noah Birnel's final project.

**Pull request closed without merging**